source/Cosmos.Core/AdvancedObject.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cosmos.Core;
+using IL2CPU.API;
+
+namespace Cosmos.Core
+{
+    /// <summary>
+    /// This Class is created for Add more methods to objects by add this to base class
```

Comment:
Please provide a better description of what kind of methods this class supports?

Reply:
I added a better description as best as i could